### ISSUE TITLE
Incorrect indentation of Twig switch statements.

### ISSUE THREAD

**Issue by eivindml** [prettydiff/prettydiff#495]
Hi,

it looks like prettydiff wrongly indents switch statements with Twig templates. Look at example below. Left hand, how it should be indented, right hand is how it actually is:

<img width="1253" alt="screen shot 2017-11-03 at 10 46 19" src="https://user-images.githubusercontent.com/202450/32367995-8b32a78a-c084-11e7-9732-d916d7723447.png">

**Comment by prettydiff** [comment]
I can reproduce this failure in the latest version of Pretty Diff

```twig
{% for %}
{% switch block.type %}
{% case "test" %}
<p>asdf</p>
{% default %}
{% endswitch %}
{% endfor %}
```

---

Out of curiosity do you know what the sudden interest in Pretty Diff's Twig support is all about?  I have had almost no feedback about Twig in the nearly 3 years I have provided support and then this week 3 valid defects related to Twig are opened.  That and this Github repo has gotten about 20 Github stars this week.

**Comment by eivindml** [comment]
Hmm. That's strange. I can say for myself that it's because [Craft CMS](http://craftcms.com) is using Twig for templating. I know Craft is gaining popularity, and a lot of design and developer studios I know of are switching from Wordpress etc. to Craft. 

But it might be something completely different.

**Comment by iamlinkus** [comment]
Hey @prettydiff, is this going to be fixed any time soon?

**Comment by DamirPecnik** [comment]
Yeah, I use GravCMS and we use TWIG in there also. Currently, use Atom with Beautify and I have issues like this... Also, Gantry 5 Framework is using the twig for Joomla, WordPress and Grav so fixing this would really be good.
